systemd-swap

```diff
@@ -431,8 +437,11 @@ case "$1" in
         systemd-notify "STATUS=Monitoring memory status..."
       }
 
+      mkdir -p $(dirname "$swapfc_path")
+      [ "${FSTYPE}" == "btrfs" ] && btrfs subvolume create "${swapfc_path}" \
+	     || mkdir "${swapfc_path}"
```

Comment:
Why try to create the subvolume / directory two times?

Reply:
Actually, I'm not really sure myself. I haven't looked in the code far enough to figure out why this line exists here. I just replaced the regular `mkdir` with the check. I guess I shhould read into the code a bit more a figure out if this part is even necessary at all